app/javascript/stylesheets/_admin.scss

```diff
@@ -176,3 +176,19 @@
     }
   }
 }
+
+.admin-public-image {
+  img {
+    max-width: 100%;
+    width: 85px;
+    height: auto;
+  }
+  max-height: 165px;
```

Comment:
Should this be on the `img` tag?

Reply:
Hey, this actually is for the public image card. I was trying to shrink them a little so that they took up a lil less space. If you don't care I can just remove it, but it's supposed to be there.

Reply:
I generally think that setting fixed heights on things is problematic.

Reply:
Ahhh. I understand now. It is already set. I don't know what to do here, we can figure it out later